`make dist` fails due to `po` file renames

This is a development files issue, rather than in the fvwm program itself.

## Expected Behaviour

That `make dist` (necessary to then move on to a Debian `dpkg-buildpackage`) should run without error and produce an appropriate archive file.

## Actual Behaviour

```
make[4]: Entering directory '/usr/local/src/fvwm3.git/doc'
make[4]: Leaving directory '/usr/local/src/fvwm3.git/doc'
make[3]: Leaving directory '/usr/local/src/fvwm3.git/doc'
 (cd po && make  top_distdir=../fvwm3-1.1.1 distdir=../fvwm3-1.1.1/po \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[3]: Entering directory '/usr/local/src/fvwm3.git/po'
make  distdir-am
make[4]: Entering directory '/usr/local/src/fvwm3.git/po'
make[4]: *** No rule to make target 'fvwm3.ar.po', needed by 'distdir-am'.  Stop.
make[4]: Leaving directory '/usr/local/src/fvwm3.git/po'
make[3]: *** [Makefile:361: distdir] Error 2
make[3]: Leaving directory '/usr/local/src/fvwm3.git/po'
make[2]: *** [Makefile:639: distdir-am] Error 1
make[2]: Leaving directory '/usr/local/src/fvwm3.git'
make[1]: *** [Makefile:633: distdir] Error 2
make[1]: Leaving directory '/usr/local/src/fvwm3.git'
make: *** [Makefile:739: dist] Error 2
```

## Extra Information

Presumably `b92ecb292ec639dc2701c3d172d6891417858d8a` didn't properly update everything for this file rename.  I'll try to track down the minimum edit in order to get this working again.  Last time I simply hardlinked the old names to the new names to get past it (Debian's build wasn't happy about symbolic links).

--- a/acinclude.m4
+++ b/acinclude.m4
@@ -1053,7 +1053,7 @@ AC_DEFUN([AM_PO_SUBDIRS],
   DUMMYPOFILES=
   for lang in $ALL_LINGUAS; do
     for dom in $ALL_DOMAINS; do
-      POFILES="$POFILES $dom.$lang.po"
+      POFILES="$POFILES $lang.po"
       GMOFILES="$GMOFILES $dom.$lang.gmo"
       UPDATEPOFILES="$UPDATEPOFILES $dom.$lang.po-update"
       DUMMYPOFILES="$DUMMYPOFILES $dom.$lang.nop"

